src/test/scripts/helm_install.sh

```diff
@@ -18,6 +18,8 @@ currentContext=$(kubectl config current-context)
 
 echo Current context: $currentContext
 
+# If this fails, you might be using an old version of Bash.
+# Update following https://itnext.io/upgrading-bash-on-macos-7138bd1066ba.
```

Comment:
Maybe we could make the script fail if the bash version is too old?

Reply:
Good idea, I made Bash 5 a requirement for this script.